README.md

```diff
@@ -1 +1,31 @@
-# angular-what-is-angular-readme
+# What is Angular and why does it exist?
+
+## Overview
+
+AngularJS (commonly referred to as just "Angular") is an open-source framework released in 2009, being mainly maintained by Google as well as a massive community of developers.
+
+It aims to solve the problems faced when creating single page applications, implementing a MVVM pattern to separate presentation and business logic.
+
+## Objectives
+
+- Describe what Angular's role in the web platform is
+- Describe the issues Angular looks to solve
+- Describe how Angular thinks in terms of Web Components to create encapsulation and reusable components
+
+## Traditional Rendering
+
+Before mainstream JavaScript usage, everything on the client was determined by the server. Routing and views were defined in backend code, meaning view logic was frequently mixed in with business logic, providing the end-user with a poor user experience.
+
+This had its disadvantages, as data displayed on the page could only be updated when the user refreshes. This also meant no smooth transitions between pages.
+
+## The role of Angular
+
+Angular brings power to the frontend, handling routing as well as all view logic. This allows frontend developers and backend developers to focus on their areas - there's no need to mix code between the two.
```

Comment:
Make should be a more general explanation of why Angular is great. Talk about using jQuery to make AJAX requests. "Angular provides us with tools and structure to easily make these types of requests, so we can focus on the interesting parts of our applications" Give a very general description of something Angular can do. "Using Angular, we can easily update our pages in real-time without refreshing." Again, you can avoid discussing routing and view logic. Keep this more general.  

Also, here you talk about not mixing code between frontend and backend developers without discussing how. In this case, I would not mention it at all. Try not to bring things up that the students may not understand without explaining how unless you have to. If you do have to bring something up, leave a note saying "we'll explain this more later, for now just know that..."